raiden/network/rpc/client.py

```diff
@@ -439,11 +439,17 @@ def _query_filter(self, function):
                 decode_topic(topic)
                 for topic in log_event['topics']
             ]
+            block_number = log_event.get('blockNumber')
```

Comment:
Yes, I supposed that, but I don't know what to put in this case, as the block number isn't yet defined. If a put higher number, it could mess with algorithms which depends on querying over the most recent block seen (i.e. webUI event pooling), that's why I put `0` here. Any idea on how it'd be better handled?